client/components/form-sections/application-table.tsx

```diff
@@ -1,110 +1,52 @@
-  const tdStyles =
-    'table-td table-header px-6 py-4 text-left text-sm font-strong border-b-2  border-bcYellowWarning';
-  return (
-    <thead className='border-b bg-bcGrayInput table-header'>
-      <tr>
-        {headers &&
-          headers.map((title: string, index: number) => (
-            <th key={`th${index}`} className={tdStyles}>
-              {title}
-            </th>
-          ))}
-      </tr>
-    </thead>
-  );
-};
-
-const TableBody: React.FC<Props> = applications => {
-  const handleSelectRow = (index: number) => {
-    alert(index);
-  };
-  const tdStyles =
-    'table-td px-6 py-4 text-left text-sm font-strong flexitems-center justify-between';
-  return (
-    <tbody>
-      {applications.applications &&
-        applications.applications.map((row: any, index: number) => (
-          <tr
-            key={`row${index}`}
-            onClick={() => handleSelectRow(row.BCAAP_Form_ID)}
-            className='bg-white border-b-2 even:bg-bcGrayInput
-              border-gray-200'
-          >
-            <td className={tdStyles}>{row.Confirmation_ID}</td>
-            <td className={tdStyles}>{row.BCAAP_Form_ID}</td>
-            <td className={tdStyles}>{row.Assigned_To}</td>
-            <td className={tdStyles}>{row.Created_At}</td>
-            <td className={tdStyles}>{row.Status}</td>
-            <td className={tdStyles}>{row.Assigned_To}</td>
-            <td className={tdStyles}>{row.Created_At}</td>
-            <td className={tdStyles}>{row.Status}</td>
-          </tr>
-        ))}
-    </tbody>
-  );
-};
-
-const ApplicationTable: React.FC<Props> = applications => {
-  return (
-    <div>
-      <table className='min-w-full text-center'>
-        <TableHeader />
-        {ApplicationTableData && applications.applications.length != 0 ? (
-          <TableBody applications={applications.applications} />
-        ) : (
-          <p className='text-center text-sm mt-4'>No applications found.</p>
-        )}
-      </table>
-    </div>
-  );
-};
-
 export const ApplicationDashboard: React.FC<any> = () => {
   const [currentPage, setCurrentPage] = useState<number>(1);
-  const [applicationsPerPage] = useState<number>(10);
-  const [searchValue, setSearchValue]: [string, (search: string) => void] = useState('');
+  const [applicationsPerPage] = useState<number>(2);
+  const [totalApplications, setTotalApplications] = useState<number>();
+  const [searchFacilityName, setSearchFacilityName] = useState<string>('');
+  const [searchConfirmatioID, setSearchConfirmatioID] = useState<string>('');
+  const [data, setData] = useState<any>([]);
 
-  const handleSearch = (e: { target: { value: string } }) => {
-    setSearchValue(e.target.value);
+  const fetchData = async (clearData: boolean) => {
+    let url;
+    if (clearData) {
+      url = `${
+        process.env.NEXT_PUBLIC_SERVER_URL
+      }/applications?facilityName=${''}&confirmationId=${''}&orderBy=status&page=${currentPage}&limit=${applicationsPerPage}`;
```

Comment:
The queryParams can remain an object and it can sent as `axios.get(url, { params: {facilityName: ''}})`

Reply:
Sounds good! Changed it.